src/Microsoft.OData.Client/Serialization/HttpClientRequestMessage.cs

```diff
@@ -8,14 +8,12 @@ namespace Microsoft.OData.Client
 { 
     using System;
     using System.Collections.Generic;

```

Comment:
I think there is more cleaning that needs to be done to this class since there are some properties like `ReadWriteTimeOut` and `Credentials` that were overridden but should be removed..

Reply:
Also remove these
https://github.com/OData/odata.net/blob/57572612f1ff833d395b44e6b2c498d7d0d31890/src/Microsoft.OData.Client/Serialization/DataServiceClientRequestMessage.cs#L63
https://github.com/OData/odata.net/blob/57572612f1ff833d395b44e6b2c498d7d0d31890/src/Microsoft.OData.Client/Serialization/DataServiceClientRequestMessage.cs#L73

Reply:
I was planning to work on these in a separate work item: #2380

Reply:
I think it's important to ensure these follow-up work items are tracked and worked on immediately, so they don't get left out.

Reply:
The item is tracked and part of ODL 8 plan. I was planning to take it up as soon as this PR is merged.